toast-tk-dao-api/src/main/java/io/toast/tk/dao/domain/impl/test/block/IProject.java

```diff
@@ -1,10 +1,11 @@
-package io.toast.tk.dao.domain.impl.test.block;
-
-import java.util.List;
-
-public interface IProject {
-
-	List<ICampaign> getCampaigns();
-
-	String getName();
-}
\ No newline at end of file
+package io.toast.tk.dao.domain.impl.test.block;
+
+import java.util.List;
+
+import io.toast.tk.dao.domain.api.repository.IElement;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.toast.tk.dao.domain.api.repository.IElement'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)